finality-aleph/src/import.rs

```diff
@@ -116,17 +126,17 @@ where
         cache: HashMap<[u8; 4], Vec<u8>>,
     ) -> Result<ImportResult, Self::Error> {
         let number = *block.header.number();
-        let hash = block.post_hash();
-        let justifications = block.justifications.take();
-
+        let post_hash = block.post_hash();
         if let Some(m) = &self.metrics {
-            m.report_block(hash, Instant::now(), "importing");
+            m.report_block(post_hash, Instant::now(), "importing");
         };
 
-        log::debug!(target: "afa", "Importing block #{:?}", number);
+        let justifications = block.justifications.take();
+
+        log::debug!(target: "afa", "Importing block {:?} {:?} {:?}", number, block.header.hash(), block.post_hash());
```

Comment:
We almost everywhere `use log::{what, ever}` and then call the macros without the namespace, could you do this here too, for consistency?

Reply:
There are a couple places this happens, just `grep -Rn log::`.

Reply:
Thanks, seems I caught a couple of them but not all. Now it is fixed everywhere.